.github/workflows/msrv.yml

```diff
@@ -86,6 +73,7 @@ jobs:
           components: rustfmt
 
       - name: Run cargo fmt
+        continue-on-error: ${{ matrix.rust == 'beta' }}
```

Comment:
This has no value if you're removing the `stable` and `beta` toolchains. Like mentioned earlier, if you plan to test only MSRV, then drop the matrix usage?